cargo-process.el

```diff
@@ -187,6 +227,26 @@ Cargo: Run the tests."
   (cargo-process--start "Test" "cargo test"))
 
 ;;;###autoload
+(defun cargo-process-current-test ()
```

Comment:
You also forgot 2 dashes here, also maybe rename this to ````cargo-process--current-fn-test```` ? Since you called the other function ````cargo-process--get-current-file-tests````. Just to be more consistent